front/app/utils/yup/validateOneOptionForMultiSelect.ts

```diff
@@ -10,15 +10,22 @@ const validateOneOptionForMultiSelect = (message: string) => {
     )
     .when('input_type', (input_type: string, schema) => {
       if (input_type === 'multiselect' || input_type === 'select') {
-        return schema.test('one-option', message, (options: IOptionsType[]) => {
-          return options
-            ? options.some((option: IOptionsType) => {
-                return Object.values(option.title_multiloc).some(
-                  (value: string) => value !== ''
-                );
-              })
-            : false;
-        });
+        return schema.test(
+          'one-option',
+          message,
+          (options: IOptionsType[], testContext: TestContext) => {
+            if (testContext.parent.key === 'topic_ids') {
```

Comment:
Since some of these yup validations are becoming a little bit specific to our use case, we might want to move them to the Form builder folders and leave the generic ones here. Not urgent for this PR. We can move them later